docs/writing-rules/experiments/join-mode/overview.md

```diff
@@ -196,7 +196,7 @@ Short for references, `refs` is a list of external rules that make up your code
 
 ### `rule`
 
-This points to an external rule location to use in this join rule. Currently, join mode requires external rules. Additionally, even though Semgrep rule files can typically contain multiple rules under the `rules` key, join mode **will only use the first rule in the file**.
+This points to an external rule location to use in this join rule. Even though Semgrep rule files can typically contain multiple rules under the `rules` key, join mode **will only use the first rule in the provided file**.
```

Comment:
```suggestion
The `rule` points to an external rule location to use in this join rule. Even though Semgrep rule files can typically contain multiple rules under the `rules` key, join mode **only uses the first rule in the provided file**.
```

Reply:
It seems a little bit redundant to me to refer back to `rule` when it's also the entirety of the heading. I like the other change, though.

Reply:
I was able to come up with a way to phrase this that's more analogous to the other sections, hoping it works!